model_compression_toolkit/target_platform_capabilities/tpc_models/imx500_tpc/v4/tp_model.py

```diff
@@ -214,6 +218,7 @@ def generate_tp_model(default_config: OpQuantizationConfig,
                         default_qco.clone_and_edit(enable_activation_quantization=False,
                                                    quantization_preserving=True)
                         .clone_and_edit_weight_attribute(enable_weights_quantization=False))
+        tp.OperatorsSet("DimensionManipulationOpsWithWeights", qpreserving_const_config_options)
```

Comment:
It's taking a lot of work to track all opsets and their differences. Maybe define them as constants in the head of the TP model with detailed comments and use them in the associated TPC.